server/Services/BookingService.cs

```diff
@@ -42,10 +40,20 @@ public async Task<ActionResult<List<BookingDto>>> GetAllFutureBookings()
         {
             return await _bookingRepository.GetAllFutureBookings();
         }
+        public async Task<ActionResult<List<BookingDto>>> GetPreviousBookingsForUser(string userId, int itemCount, int pageNumber)
+        {
+            return await _bookingRepository.GetPreviousBookingsForUser(userId, itemCount, pageNumber);
+        }
+
+        public async Task<int> GetPreviousBookingCountForUser(string userId)
+        {
+            return await _bookingRepository.GetPreviousBookingCountForUser(userId);
+        }
+
 
-        public async Task<ActionResult<List<BookingDto>>> GetAllBookingsForUser(String userId)
+        public async Task<ActionResult<List<BookingDto>>> GetActiveBookingsForUser(String userId)
```

Comment:
Tror det er bedre å kalle det GetAllBookingsForUser? vi har ikke noe attribute å vise om den booking er aktive

Reply:
UpcomingBookingForUser?

Reply:
Den henter alle bookinger som er fremdeles gjeldene. Den skal derfor ikke ta med bookings fra tidligere dager. Om den heter GetAllBookingsForUser virker det som den gir både aktive (fremtidige) og gamle bookings. Vi burde vell kunne skille mellom aktive og ikke aktive bookings i backend?

Reply:
![image](https://github.com/omegapointnorge/omega-office-booking/assets/57482883/095d1af7-9039-44e3-949c-95bf453e0b62)
Tanken var om bookingen er for i dag eller i fremtiden blir den satt som aktiv

Reply:
> UpcomingBookingForUser?

Den har blitt endret til ActiveBookings : )